scripts/remove_flag_file_name.py

```diff
@@ -0,0 +1,23 @@
+import yaml
```

Comment:
I assume this made things much easier when doing it :D
Not sure we really need to keep the script though?

Reply:
Considering there was 250+ files to look though it sure did 😅

But you are right, we don't need to keep the script. I'll push a commit that removes it.